src/components/task/existing-card/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import propTypes from "prop-types";
+import dayjs from "dayjs";
+
+import Button from "../../base/button";
+import Image from "../../base/image";
+import Styles from "./index.module.scss";
+import TextArea from "../../base/text-area";
+
+function TaskCard({
+  title,
+  createdAt,
+  isCompleted,
+  completedAt,
+  isTaskOnEditMode,
+}) {
+  const [isTextAreaVisible, setIsTextAreaVisible] = useState(isTaskOnEditMode);
```

Comment:
ditto

Reply:
**isTextAreaVisible** flag solely sets the visibility of text area inside the **task-card** only. So I feel that transferring this state to redux won't be good move @fza-wtag vai